src/App.tsx

```diff
@@ -1,61 +1,116 @@
+import { useState, FormEvent } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { getUserByID, getNewTodoId } from './utils';
+
+const todosWithUser = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserByID(usersFromServer, todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosWithUser);
+  const [title, setTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
+  const [userId, setUserId] = useState(0);
+  const [userError, setUserError] = useState(false);
+
+  const handleTitleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Consider using a more descriptive name for the event parameter, such as `event`, to improve code readability and maintainability. This aligns with the checklist item: "Don't use one-letter variable naming."